packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -40,7 +40,7 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
   private _calloutId: string;
```

Comment:
You also need to create a change file for this fix. See https://github.com/microsoft/fluentui/wiki/Setup#making-a-pull-request on instructions.